hearts/static/room.js

```diff
@@ -7,15 +7,35 @@ function apiCard(displayCard) {
     return displayCard[1] + displayCard[0];
 }
 
-// Will remove dummy data once we pull state from the API
+// API returns data of the form
+/*
+{
+    'players': [str],
+    'max_points': int,
+    'rounds': [
+        {
+            'players': str,
+            'direction': str,
+            'turn': int,
+            'hands': {str: [str]},
+            'tricks': [[str]],
```

Comment:
Currently, a serialized trick is in the form `{str: {int: str}}`  (cf. https://github.com/j2kun/lonely-hearts/blob/master/hearts/game/hearts.py#L545-L554)

I don't remember why I made it that way.  A trick really should simply be a list of tuples `[(player_string, card_string)]`  Is this worth refactoring?